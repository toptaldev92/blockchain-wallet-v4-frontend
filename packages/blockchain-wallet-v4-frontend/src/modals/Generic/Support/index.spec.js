import { shallow } from 'enzyme'
import { Support } from './index'
import React from 'react'
import toJson from 'enzyme-to-json'

describe('Support modal component', () => {
  it('matches snapshot', () => {
    const component = shallow(<Support />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
