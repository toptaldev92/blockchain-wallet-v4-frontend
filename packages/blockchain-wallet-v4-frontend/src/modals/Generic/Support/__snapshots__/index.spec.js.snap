// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Support modal component matches snapshot 1`] = `
<Modal
  size="small"
>
  <styled.div
    data-e2e="swapGetStarted"
  >
    <Styled(Text)
      size="20px"
    >
      <FormattedHTMLMessage
        defaultMessage="Need some help?"
        id="modals.support.need_some_help"
        values={Object {}}
      />
    </Styled(Text)>
    <Styled(Text)>
      <FormattedMessage
        defaultMessage="Our Blockchain Support Team is standing by."
        id="modals.support.our_support"
        values={Object {}}
      />
    </Styled(Text)>
    <Styled(Link)
      href="https://blockchain.zendesk.com/hc/en-us/requests/new?ticket_form_id=360000180551"
      target="_blank"
    >
      <Styled(Button)
        fullwidth={true}
        nature="primary"
        size="20px"
      >
        <FormattedMessage
          defaultMessage="Contact Support"
          id="modals.support.contact_support"
          values={Object {}}
        />
      </Styled(Button)>
    </Styled(Link)>
    <Styled(Image)
      name="identity-verification"
    />
  </styled.div>
</Modal>
`;
